### PR TITLE
GitHub 2 Edux

### PR DESCRIPTION
**Pull request by hroncok**
The list of files to convert and upload is specified manually and i would like to keep it that way for a while.

Also, please don't abuse my Edux credentials :D

### CODE REVIEW

File: g2e.py [hunk 1]
```diff
@@ -0,0 +1,84 @@
+"""Uploads stuff from GitHub to Edux"""
+import io
+import os
+import pathlib
+import re
+import subprocess
+
```
Comment (syky27) [inline g2e.py:7]: Delete this line

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  email: false
+deploy:
+  provider: script
+  script: python g2e.py
+  on:
+    branch: master
+    repo: 3DprintFIT/BI-3DT
+env:
+  global:
+  - secure: pNqZ7oB8D3vnFK+k9Ud4/Vw21NQh7zGn49X9pyi/1qB1xMcSj7rCvN+OiN+myAjQ7DtZYHBdrpF7KMfq+FTDbqKRhEA1MBj7dE3GoScCXNPMTGudZE1k6JfWRvMqEYe5AmGoK4gTupq4ZVq/1wF35SZyvJ5NfoI/BaVlYOFvJeJGUTx56itusWQedJrdZChVcaT4UgwGh7cZokTwysR2upfAJfivkH5cTeNo0yk6aBGpd/lnk+WwjA79K5p1lh6KC0RHwL+ZK0ryeZf1lgEYC6dsqi/KQov23rU2Stb15cRb+n8Zx9jD4FzHQo8dfPTYcWs9E98RdbXYuARrM33ozut60Kmxairm/eDfujtA1/tLpkjmeqiFrsS/v83cLsk6To/bbbKFW2NCH56jM/wMHRKZ3pIPzbor3nonLf+UR5ivfjaUDqVPf02tUT8fab8bFy2XF6jU8MMlDFioMINkc/ATami5MIyyrWu+mAjxn9fdk19vgzQpq+kCdR1TCkmuaO5irGihsrZUxCuuXlk9B5YBjYN+qEDe4qsC0B5Wnym1EtNMSeT6DqrdZFiMWu6xs4F/zdmAj1XZ2/ilp2Au3XzWonank9/sgn1xy5H0fAFkKYV/NJckrw8t3JSwU3ool57yfm+dO6/SkU9sOMYkyG2Y05bunmRq4cVyuWdTuuQ=
```
Comment (syky27) [inline .travis.yml:24]: What are these strings?

**Review by syky27 — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 2 above)
Comment (hroncok) [reply .travis.yml:24]: the env used in the script, my Edux credentials :)

File: g2e.py (hunk 1 above)
Comment (hroncok) [reply g2e.py:7]: why?

File: .travis.yml (hunk 2 above)
Comment (syky27) [reply .travis.yml:24]: This is a public repo, get it out of here, you can use Travis web UI to insert secret tokens.

File: g2e.py (hunk 1 above)
Comment (hroncok) [reply g2e.py:7]: (it splits standard library modules from pip installed ones)

Comment (syky27) [reply g2e.py:7]: Well, why is it there in first place? I just thought, why not to make it nicer... :)

File: .travis.yml (hunk 2 above)
Comment (hroncok) [reply .travis.yml:24]: this is encrypted :) I use it all the time

File: g2e.py (hunk 1 above)
Comment (syky27) [reply g2e.py:7]: Aha, okay make sense... 👍

File: .travis.yml (hunk 2 above)
Comment (hroncok) [reply .travis.yml:24]: the only 2 security problmes are:

 * Travis Ci knows my credentials
 * anyone with write access to this repo can do whatever they want with my credentials

(both threads are also present when the web UI is used to set those)

Comment (syky27) [reply .travis.yml:24]: as long as someone without write access cannot use the knowledge gain by reading this repo to abuse your creds you are good to go...

**Review by syky27 — APPROVED** [review]

File: .travis.yml (hunk 2 above)
Comment (hroncok) [reply .travis.yml:24]: https://docs.travis-ci.com/user/encryption-keys/

**Comment by hroncok** [comment]
Let me merge this

**Comment by hroncok** [comment]
And, it works :tada:

**Comment by hroncok** [comment]
I've enabled **Auto cancel branch builds** and I've set **Limit concurrent jobs** to 1. Just to make sure.

**Pull request merged**